drivers/hmis/lib/form_data/allegheny/assessments/base_annual.json

```diff
@@ -1,19 +0,0 @@
-{
-  "item": [
-    {
-      "fragment": "#information_date"
-    },
-    {
-      "fragment": "#income_and_sources"
-    },
-    {
-      "fragment": "#non_cash_benefits"
-    },
-    {
-      "fragment": "#health_insurance"
-    },
-    {
-      "fragment": "#disability"
```

Comment:
We no longer need this override, because the point of it was to include disability at annual. Now we do that by default.